manifest-generation/runtime-aci-tile.yml

```diff
@@ -0,0 +1,412 @@
+---
+releases: 
+- name: aci-containers
+  file: aci-containers-release-0.0.9.tar.gz
+  version: '0.0.9'
+
+
+stemcell_criteria:
+  os: ubuntu-trusty
+  version: '3468.21'
+  default: true  
+  enable_patch_security_updates: true
+
+property_blueprints:
+- name: aci_db_admin_password
+  type: secret
+  configurable: false
+- name: uaa_clients_aci_cf_controller_secret
+  type: secret
+  configurable: false
+- name: generated_pas_client_cert
+  type: rsa_cert_credentials
+  default:
+    domains:
+    - cf-etcd.service.cf.internal
```

Comment:
Not sure I understand why   pas_client_cert is generated with etcd in the domain name